src/Command/Project/ProjectCreateCommand.php

```diff
@@ -216,6 +262,40 @@ protected function getAvailableRegions($runtime = false)
         return $regions;
     }
 
+    /**
+     * Return the catalog.
+     *
+     * The default list is in the config `service.catalog`. This is
+     * replaced at runtime by an API call.
+     *
+     * @param bool $runtime
+     *
+     * @return array
+     */
+    protected function getDefaultCatalog($runtime = false)
+    {
+        if ($runtime) {
+            $catalog = [];
+            $catalog_items = $this->api()->getClient()->getCatalog()->getData();
+            if (!empty($catalog_items)) {
+                foreach ($catalog_items as $item) {
+                    if (isset($item['info']) && isset($item['template'])) {
+                        $catalog[$item['template']] = $item['info']['name'];
+                    }
+                }
+            }
+            $catalog['empty'] = 'Empty Project';
```

Comment:
It seems you want this always to be an option in `$catalog` whether it comes from config or from the remote catalog

Reply:
I'm really not sure about this, so I removed it. Now that we're falling back on to the `setup/options` api this is probably a non-issue.